src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -61,8 +61,8 @@ public class Arm extends SubsystemBase {
     private Rotation2d handAngle = new Rotation2d();
     private Rotation2d handAngleToGround = new Rotation2d();
 
-    private final double SETPOINT_POSITION_TOLERANCE = Units.degreesToRadians(1);
-    private final double SETPOINT_VELOCITY_TOLERANCE = Units.degreesToRadians(5);
+    private final double SETPOINT_POSITION_TOLERANCE = 2.5;
```

Comment:
this is in radians -.- where did the degreesToRadians go?

Reply:
supposedly, however this made things have intended behavior

Reply:
fine... ill look at it later